job-service-acceptance-tests/src/test/java/com/hpe/caf/jobservice/acceptance/FinalOutputDeliveryHandler.java

```diff
@@ -38,7 +38,7 @@
 /**
  * Verifies result messages issued at the end of the end-to-end test.
  */
-public class FinalOutputDeliveryHandler implements ResultHandler {
+public class FinalOutputDeliveryHandler<T extends TaskMessage> implements ResultHandler<T> {
```

Comment:
Why are you making `FinalOutputDeliveryHandler` generic instead of just implementing `ResultHandler<TaskMessage>`?

Reply:
Updated